lib/Test/Pod/CoverageChange.pm

```diff
@@ -0,0 +1,246 @@
+package Test::Pod::CoverageChange;
+# ABSTRACT: check perl modules against their pod coverage
+
```

Comment:
strict, warnings and POD encoding should always be at the top of the file. Would also need the $VERSION definition.

```
use strict;
use warnings;

our $VERSION = '0.001';
# AUTHORITY

use utf8;

=encoding utf8
```